aptos-move/aptos-e2e-comparison-testing/src/data_collection.rs

```diff
@@ -213,27 +239,26 @@ impl DataCollection {
                     let dump_write_set = self.dump_write_set;
                     let data_manager = data_manager.clone();
                     let index = index_writer.clone();
-
-                    let state_view =
-                        DataStateView::new_with_data_reads(self.debugger.clone(), version);
+                    let base_experiments = base_experiments.clone();
+                    let data_state = InMemoryStateStore::default();
+                    let features_to_enable = vec![FeatureFlag::ENABLE_ENUM_OPTION];
```

Comment:
Why is that? Adding a comment can be helpful?

Reply:
added the comment